catalyst-gateway/bin/src/service/api/health/live_get.rs

```diff
@@ -10,6 +10,10 @@ pub(crate) enum Responses {
     /// Service is OK and can keep running.
     #[oai(status = 204)]
     NoContent,
+    /// Service is possibly not running reliably.
+    #[oai(status = 503)]
+    #[allow(dead_code)]
+    ServiceUnavailable,
```

Comment:
Ok, the logic here is also needs expansion.
Similar to "started" we should have a global state variable called "ready" which defaults to true. 
A function (which can be marked "dead_code" for now, until its used, can set that variable to false.
The endpoint then return NoContent if its true, and ServiceUnavailable otherwise.